python/lsst/ts/standardscripts/auxtel/offset_ataos.py

```diff
@@ -0,0 +1,155 @@
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/auxtel/offset_atcs.yaml
+            title: OffsetATCS v1
+            description: Configuration for OffsetATCS Script.
+            type: object
+            properties:
+              z:
+                type: number
+                description: Offset hexapod in z axis (mm).
+              x:
+                type: number
+                description: Offset hexapod in x axis (mm).
+              y:
+                type: number
+                description: Offset hexapod in y axis (mm).
+              u:
+                type: number
+                description: Rx offset (deg).
+              v:
+                type: number
+                description: Ry offset (deg).
+              m1:
+                type: number
+                description: M1 pressure offset (Pa).
+                maximum: 0
+              offset_telescope:
+                type: bool
+                description: When correcting coma, also offset the telescope?
+                type: boolean
+                default: true
+              reset_offsets:
+                type: array
+                description: >-
+                    List of axes to reset. Must be one of z, x, y, z, u, v,
+                    m1, all
+                items:
+                    type: string
+
+            additionalProperties: false
+            anyOf:
+                - required: ["z"]
+                - required: ["x"]
+                - required: ["y"]
+                - required: ["u"]
+                - required: ["v"]
+                - required: ["m1"]
+                - required: ["reset_offsets"]
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+
+        self.offset_z = getattr(config, "z", 0.0)
+        self.offset_x = getattr(config, "x", 0.0)
+        self.offset_y = getattr(config, "y", 0.0)
+        self.offset_u = getattr(config, "u", 0.0)
+        self.offset_v = getattr(config, "v", 0.0)
+        self.offset_m1 = getattr(config, "m1", 0.0)
+        self.reset_offsets = getattr(config, "reset_offsets", None)
+
+        self.offset_telescope = config.offset_telescope
+
+        self.offsets = {
+            "z": self.offset_z,
+            "x": self.offset_x,
+            "y": self.offset_y,
+            "u": self.offset_u,
+            "v": self.offset_v,
+            "m1": self.offset_m1,
+        }
+
+    def set_metadata(self, metadata):
+        metadata.duration = 10
+
+    async def assert_feasibility(self) -> None:
+        """Verify that the telescope is in a feasible state to
+        execute the script.
+        """
+
+        await self.atcs.assert_all_enabled()
+
```

Comment:
We should also assert that the ataos corrections are enabled. 

In the track_target script we do [this](https://github.com/lsst-ts/ts_standardscripts/blob/05ad34eee66b7e7b3f29bf6ea5b301cdd9ce0e36/python/lsst/ts/standardscripts/auxtel/track_target_and_take_image.py#L236-L251), but we should probably move this to `ATCS` and call it here.

Reply:
I will add the standard block of commands here and the new ATCS method in a separate ticket https://jira.lsstcorp.org/browse/DM-39297

Reply:
Please, add a "TODO" comment here. No need to update the other instances where this occurs, but let's have it at least here, e.g.;

```
# TODO (DM-39297): Replace this with call to assert ataos corrections are enabled in ATCS.
```